.github/workflows/android.yml

```diff
@@ -0,0 +1,25 @@
+name: Android CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - uses: gradle/wrapper-validation-action@v1.0.4
+    - name: set up JDK
+      uses: actions/setup-java@v2
+      with:
+        java-version: 11
+        distribution: 'temurin'
```

Comment:
not sure what the diff between temurin and zulu is.  I see zulu in a few places.

Reply:
yeah, i was trying them all when trying to get a test to work in android-times-square.  i'll switch to zulu, shouldn't matter here